File: 58_Length_Of_Last_Word.java
class Solution {
    public int lengthOfLastWord(String s) {
        if(s.length() <= 0) return 0; 
        s = s.trim(); 
        int rightPointer = s.length() - 1; 
        int count = 0; 
        while(rightPointer >= 0)
        {
            if(s.charAt(rightPointer) == ' ')
            {
                return count; 
            }
            count++; 
            rightPointer--; 
        }
        return count; 
    }
}

File: 58_Length_Of_Last_Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        if len(s) <= 0:
            return 0 
        s = s.rstrip() 
        count = 0 
        rightPointer = len(s) - 1 
        while(rightPointer >= 0):
            if s[rightPointer] == ' ': 
                return count 
            rightPointer -= 1 
            count += 1 
        return count 